applications/fishing-map/src/features/datasets/datasets.utils.ts

```diff
@@ -18,6 +19,13 @@ export const removeDatasetVersion = (datasetId: string) => {
   return datasetId ? datasetId?.split(':')[0] : ''
 }
 
+export const getDatasetLabel = (dataset: { id: string; name?: string }): string => {
+  const { id, name = '' } = dataset || {}
+  if (!id) return name || ''
+  const label = getDatasetNameTranslated(dataset)
+  return dataset.id.includes(PRIVATE_SUFIX) ? `${label} 🔒` : label
```

Comment:
Why displaying an emoji when we have a good-looking icon? 
--> https://github.com/GlobalFishingWatch/frontend/pull/815/files#diff-e1152028eb6a3aeb8f5cd07e285f5f727f76aff26dd2dfe1f9f8d48eb15b124bR60

Reply:
This method could optionally return a React element when it's possible to use one? 
It would be possible, for instance, here: https://github.com/GlobalFishingWatch/frontend/pull/815/files#diff-3145ce8092c05120599c3ac73fd565cb66ad5ca0f456527adeed3988ad638bb1R233

When it's not, I'd much rather have `${label} (private)`, than the emoji

Reply:
I tried first to use the <Icon /> component but there are many places where the component API supports only a string, for example:
- [`<TagList />`](https://github.com/GlobalFishingWatch/frontend/blob/fishing-map/private-datasets/packages/ui-components/src/tag-list/index.ts#L8) used in sources filter layer panel:
![image](https://user-images.githubusercontent.com/10500650/130058867-5c170d7e-caeb-4c51-b0c7-28ac83f9eba7.png)

- [`<MultiSelect />`](https://github.com/GlobalFishingWatch/frontend/blob/fishing-map/private-datasets/packages/ui-components/src/multi-select/index.ts#L6) used in the search:
![image](https://user-images.githubusercontent.com/10500650/130058962-c56e6cfe-e6f0-45c4-9a68-27278a94eaee.png)

So we thought this was the easier way to include it as we can keep as plain text. 
But we could support ReactElements if we think this can be useful in the future, WDYT ?

Reply:
I think that would be worth it and not too hard to do (only a type change in the best case scenario)

Reply:
Sounds reasonable, let me try it and will let you know!